docs/source/pages/documentation.md

```diff
@@ -443,173 +719,347 @@ When specifying the names of subjects or sessions to transfer, we often want to
             └── behav
 ```
 
-We can use the wildcard tag to match everything that comes after the `date` key. For example, to upload the these sessions from *local* to *central*, we can perform:
-```
+We can use the wildcard tag in the *session* name to match
+everything that comes after the `date` key:
+
+::::{tab-set}
+
+:::{tab-item} macOS / Linux
+```{code-block} console
 datashuttle \
 my_first_project \
 upload \
--sub 001 002
--ses 005_condition-test_date-@*@
+-sub 001 002 \
+-ses 005_condition-test_date-@*@ \
 -dt behav
 ```
+:::
+
+:::{tab-item} Windows
+```{code-block} console
+datashuttle ^
+my_first_project ^
+upload ^
+-sub 001 002 ^
+-ses 005_condition-test_date-@*@ ^
+-dt behav
+```
+:::
 
-Which would selectively upload session 5 from subjects 1 and 2.
+:::{tab-item} Python API
+```{code-block} python
+project.upload(
+	sub_names=["001", "002"],
+	ses_names="005_condition-test_date-@*@",
+	datatype="behav",
+)
+```
+:::
 
-If using macOS (or, in general, the `z-shell (zsh)`, names including the `@*@` flag must be wrapped in quotation marks, for example `--ses "005_condition-test_date-@*@"`.
+::::
 
-Similarly, if the test session was on different session numbers for different sessions, we could use `-ses @*@condition-test@*@` (as the `-ses` argument above) to selectively transfer test sessions only.
+This command would transfer session 5 from subject 001 and 002.
 
+::: {warning}
+
+On macOS (or with the z-shell (zsh) in general), text
```

Comment:
Is it a Mac problem or a zsh problem? I expect the latter. While zsh is the default on Mac, it may be changed.
Perhaps phrase as "If using the z-shell (zsh) - which is the default shell on MacOS - ..."